common/leica_sep.c

```diff
@@ -59,24 +60,29 @@ static int leica_sep_transfer(struct leica_sep_funcs *f, char *cmd,
 		if (data[bytes_received] == '\n' &&
 		    bytes_received >= 1 &&
 		    data[bytes_received - 1] == '\r') {
-			bytes_received--;
-
-			if (bytes_received >= 2 &&
-			    data[bytes_received - 2] == '"')
+			if (header_received) {
```

Comment:
Does this mean that the "unexpected" packets will necessary have a header?
Otherwise we continue to read till the end of the timeout and return anyway -1, because `command_received` is still false.

Reply:
Yes - they have a header (since it is SEP) and even if they wouldn't, we just set `bytes_received` to zero if `\r\n` is received and continue with waiting for our header